### PR TITLE
add burn-out content

### PR DESCRIPTION
**Pull request by dawedawe**

### CODE REVIEW

File: src/pages/burn-out.md [hunk 1]
```diff
@@ -7,12 +7,28 @@ color: "yellow"
 
 ## Intro
 
-Maintainers can get burned-out.
+Let's not kid ourselves. Open Source can be a miserable place to work in.
```
Comment (nojaf) [inline src/pages/burn-out.md:10]: The spelling of `Open Source` seems a bit off.
Should it be like this? Or `Open-source`?

File: src/pages/burn-out.md [hunk 2]
```diff
@@ -7,12 +7,28 @@ color: "yellow"
...
 
-Maintainers can get burned-out.
+Let's not kid ourselves. Open Source can be a miserable place to work in. 
+After a full work day random strangers expect you to perform at your best, for free, for them. 
+And if you don't, they will let you know. 
+
+There are many toxic social dynamics unique to Open Source that can drain all energy from you.  
+
+But it's not just the human factor. It's also the technical challenges that can be overwhelming. 
+Often you find yourselve facing the decision to either go to bed and get a healthy amount of sleep, or to stay up late and fix that bug, to read up on that API change that made your CI crash, to fix that issue that has been open for months, etc. etc. etc.
```
Comment (nojaf) [inline src/pages/burn-out.md:17]: `etc. etc. etc.` A single `etc` should suffice.

File: src/pages/burn-out.md [hunk 3]
```diff
@@ -7,12 +7,28 @@ color: "yellow"
...
 
-Let's catch up!
+There are no silver bullets. Leaving Open Source is not the answer most of you are looking for. It's not the answer we turned to in the past.  
+But there are things we can do together to make things better.  
+To make expectations clear: We don't offer help maintaining your project. We don't offer help with your technical problems beyond that what AF does in general. We don't offer help with your personal problems.  
+
+All we offer is a place to talk about your situation with people who have the experience to really understand what you're going through. 
+We stand by our word that we will not share anything you tell us with anyone else.  
+
+So if this offer sounds interesting to you, please reach out to us. To us as a group, or to one of us individually. What ever you feel most comfortable with.
```
Comment (nojaf) [inline src/pages/burn-out.md:34]: Mention the gmail email address here.

**Review by nojaf — COMMENTED** [review]
Great start!

File: src/pages/burn-out.md (hunk 1 above)
Comment (dawedawe) [reply src/pages/burn-out.md]: Hard to say what's "correct", you find all kinds of spellings, even in our page.
I went with "Open-Source", but no hill I want to die on 😆

**Review by nojaf — APPROVED** [review]
Thanks!

File: src/pages/burn-out.md (hunk 1 above)
Comment (nojaf) [reply src/pages/burn-out.md]: Yeah, I struggle with this one a lot myself as well.

**Review by edgarfgp — APPROVED** [review]

**Pull request merged**